contrib/epee/include/net/abstract_tcp_server2.h

```diff
@@ -49,10 +49,12 @@
 #include <boost/asio/ssl.hpp>
 #include <boost/array.hpp>
 #include <boost/noncopyable.hpp>
-#include <boost/shared_ptr.hpp>
+#include <boost/shared_ptr.hpp> //! \TODO Convert to std::shared_ptr
```

Comment:
Is this still needed? code looks to be using std::shared_ptr.

Reply:
There are still some `boost::shared_ptr`'s in this file, so I added this comment.

Reply:
Gotcha